ee/localserver/krypto-ec-middleware.go

```diff
@@ -61,13 +61,33 @@ func (cmdReq v2CmdRequestType) CallbackReq() (*http.Request, error) {
 	return req, nil
 }
 
+// kryptoEcMiddlewareResponse is a box that contains the response, hardware signature of the response,
+// and the public key of the hardware signer.
+type kryptoEcMiddlewareResponse struct {
+	// To validate the response.
+	// 1. Convert th b64der hardware key field to public key
+	// 2. Determine if hardware key is trusted
+	// 3. B64 decode message and hardware sig to bytes
+	// 4. Use decoded msg, hardware sig, and public key to verify the signature
+	// 4. After verifying the signature, unmarshall the b64 decoded msg to json
+
+	// Msg is the base64 encoded message that contains timestamp, nonce, and
+	// the response from the endpoint called.
+	Msg string `json:"msg"`
```

Comment:
not sure if this is the best way to pacakge this data, maybe we should continue to return the endpoint repose as it is and take on a `HardwareSignatureBox` something like:

```go
type kryptoEcMiddlewareResponse2 struct {
	Data string `json:"data"`
	// HardwareSigBox is b64 encoded []byte of msgpack containing
	// msg, sig, and key []byte fields
	HardwareSigBox string `json:"hardware_sig_box"`
}

type hardwareSigBox struct {
	Msg    []byte `msgpack:"msg"`
	Sig    []byte `msgpack:"sig"`
	PubKey []byte `msgpack:"pub_key"`
}
```